src/Test.UnitTests.Sarif/Baseline2/V2ResultMatcherTests.cs

```diff
@@ -347,12 +349,20 @@ public void ResultMatchingBaseliner_WhenThereIsOnlyOneCurrentRun_CopiesSelectedR
                     new Result
                     {
                         RuleId = TestConstants.RuleIds.Rule1,
-                        RuleIndex = 0
+                        RuleIndex = 0,
+                        Provenance = new ResultProvenance
+                        {
+                            FirstDetectionTimeUtc = firstDetectionTime
+                        }
                     },
                     new Result
                     {
                         RuleId = TestConstants.RuleIds.Rule2,
-                        RuleIndex = 1
+                        RuleIndex = 1,
+                        Provenance = new ResultProvenance
+                        {
+                            FirstDetectionTimeUtc = firstDetectionTime
```

Comment:
>FirstDetectionTimeUtc [](start = 28, length = 21)

Now that the result matcher adds `firstDetectionTimeUtc` to the current result, we must place it on the previous result as well or this test will fail, because it performs a naive property-by-property comparison of the current to the previous result (using `ResultEqualityComparer`).